deploy/templates/gcs-manifests/gcs-csi.yml.j2

```diff
@@ -164,16 +164,21 @@ spec:
           args:
             - "--v=5"
             - "--csi-address=$(ADDRESS)"
+            - "--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"
           env:
             - name: ADDRESS
               value: /plugin/csi.sock
+            - name: DRIVER_REG_SOCK_PATH
+              value: /var/lib/kubelet/plugins/org.gluster.glusterfs/csi.sock
```

Comment:
If we really do need a separate variable for the registration socket, then shouldn't the value here be `/registration/org.gluster.glusterfs/csi.sock`?

Reply:
the driver-registrar expose a Unix domain socket to handle Kubelet Plugin Registration.
 this socket MUST be surfaced on the host in the kubelet plugin registration directory (in addition to the CSI driver socket)